index.html

```diff
@@ -49,6 +49,50 @@ <h2 class="section-box__title">Add product</h2>
       <section class="
               section-box">
         <h2 class="section-box__title">Today - 06.01.2019</h2>
+
+        <div class="section-box__wrapper">
+
+          <div class="section-box__wrapper--left">
+            <h3 class=section-box__title--left>Choosen products:</h2>
+              <ul class="product-list product-list--choosen-products">
+                <li class="product-list__item product-list__item--choosen-products">
+                  <div class="product-list__title ">Apple</div>
+                  <small class="product-list__description">Amount: 150g</small>
+                  <small class="product-list__description">results: 78kcal</small>
+                </li>
+                <li class="product-list__item product-list__item--choosen-products">
+                  <div class="product-list__title">Egg</div>
+                  <small class="product-list__description">Amount: 100g</small>
+                  <small class="product-list__description">results: 140kcal</small>
+                </li>
+                <li class="product-list__item product-list__item--choosen-products">
+                  <div class="product-list__title">Banana yoghurt</div>
+                  <small class="product-list__description">Amount: 200g</small>
+                  <small class="product-list__description">results: 172kcal</small>
+                </li>
+                <li class="product-list__item product-list__item--choosen-products">
+                  <div class="product-list__title">Chicken and rice</div>
+                  <small class="product-list__description">Amount: 100g</small>
+                  <small class="product-list__description">results: 79kcal</small>
+                  </small>
+                </li>
+                <li class="product-list__item product-list__item--choosen-products">
+                  <div class="product-list__title">Orange</div>
+                  <small class="product-list__description">Amount: 500g</small>
+                  <small class="product-list__description">results: 395kcal</small>
+                </li>
+              </ul>
+          </div>
+
+          <div class="section-box__wrapper--right">
+            <h3 class="section-box__title--right">Total:</h3>
+            <div class="total__label--wrapper">
+              <label class="total__label">You ate x calories today.</label>
+              <label class="total__label--description">See you tomorrow.</label>
+            </div>
+            <img class="section-box__image" src="http://mjsw.gis.gov.pl/Resources/192/kolowy.jpg">
```

Comment:
Ten wykres nie wnosi niczego prócz kwestii dekoracyjnych, można usunąć. Na samym końcu tego projektu być może zajmiesz się generowaniem takiego wykresu na podstawie prawdziwych danych albo rozwiążemy to inaczej ;)